README.md

```diff
@@ -35,131 +35,54 @@
 # After
 
 + BotD is a browser library for JavaScript bot detection
-+ Easily add ability to detect automation tools, browser spoofing and virtual machines
-+ Requires adding only 3 lines of JavaScript on your website
++ Easily add ability to detect automation tools
++ Requires adding only a few lines of JavaScript on your website
 ```
 
-### 🔩 [Try Demo](https://fingerprint.com/products/bot-detection/) - see the live demo running in your browser
+#### 🔩 [Try Demo](https://fingerprint.com/products/bot-detection/) - see the live demo running in your browser
```

Comment:
I suppose this link should point to http://fingerprintjs.github.io/BotD, because fingerprint.com will host a version of BotD not from this repository.